### ISSUE TITLE
Remote sources do not seem to work

### ISSUE THREAD

**Issue by DemoMacro** [nuxt/content#1921]
### Environment

❯ npx nuxt info
Nuxi 3.2.2

RootDir: /home/projects/nuxt-content-remote-source
Nuxt project info: 

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.2.2`
- Nitro Version:    `2.2.3`
- Package Manager:  `npm@7.17.0`
- Builder:          `vite`
- User Config:      `modules`, `content`
- Runtime Modules:  `@nuxt/content@2.4.3`
- Build Modules:    `-`
------------------------------

👉 Report an issue: https://github.com/nuxt/nuxt/issues/new

👉 Suggest an improvement: https://github.com/nuxt/nuxt/discussions/new

👉 Read documentation: https://nuxt.com

### Reproduction

https://stackblitz.com/edit/nuxt-content-remote-source?file=nuxt.config.ts

### Describe the bug

I tried to use the remote source function of `@nuxt/content` but it doesn't seem to work, is there something wrong with my configuration? Or is it something else? The description in the documentation is really vague.
```
export default defineNuxtConfig({
  modules: ['@nuxt/content'],
  content: {
    sources: {
      // Additional sources
      contentnew: {
        prefix: '/contentnew', // All contents inside this source will be prefixed with `/contentnew`
        driver: 'http',
        // ...driverOptions
        base: 'https://cdn.jsdelivr.net/gh/nuxt/content@2.4.3/docs/content/', // Path for source directory
      },
    },
  },
});
```
Only the following error is reported.
```
Document not found, overwrite this content with #not-found slot in <ContentDoc>.
```

**Comment by Barbapapazes** [comment]
Yep, I don't know how to use it but I'm very interested by this feature. Maybe some docs could be great!
